docs/x509.rst

```diff
@@ -181,6 +181,71 @@ X.509 Certificate Object
 
         For version 3 X.509 certificates.
 
+.. class:: Attribute
+
+    .. versionadded:: 0.8
+
+    .. attribute:: oid
+
+        :type: :class:`ObjectIdentifier`
+
+        The attribute OID.
+
+    .. attribute:: value
+
+        :type: :class:`str`
```

Comment:
str on py2 is bytes, not text. `unicode` is text. Do we have any other
places that do this? How do we document them?

On Fri, Jan 30, 2015 at 8:23 AM, Paul Kehrer <notifications@github.com>
wrote:

> In docs/x509.rst
> <https://github.com/pyca/cryptography/pull/1645#discussion_r23841641>:
>
> > @@ -181,6 +181,71 @@ X.509 Certificate Object
> >
> >          For version 3 X.509 certificates.
> >
> > +.. class:: Attribute
> > +
> > +    .. versionadded:: 0.8
> > +
> > +    .. attribute:: oid
> > +
> > +        :type: :class:`ObjectIdentifier`
> > +
> > +        The attribute OID.
> > +
> > +    .. attribute:: value
> > +
> > +        :type: :class:`str`
>
> I made them text since they're a dotted representation, so they're a str
> on py2 and py3.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1645/files#r23841641>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084